src/App.js

```diff
@@ -1,64 +1,59 @@
-import React, { Component } from 'react';
+/*
+   Author: Michael Babko
+   Date: March 2021
 
-/**
- * App class runs entire application
- */
-class App extends Component {
+   Renders the page with a live Todolist.
+*/
+   import React, { useState } from 'react';
+   
+   import Alert from 'react-bootstrap/Alert';
+   import ManageTodolist from './components/ManageTodolist';
+   
    /**
-   * Represents a book.
-   * @constructor
-   */
-   constructor() {
-      super();
-   }
+    * Sends POST request to Todolist API to add a Todolist item.
```

Comment:
-1 Incorrect Comment, the App function specifically doesnt do any POST requests. 
You had the correct comment up above as a file header comment which is a good.

It is important to not have any comments that misdirect or confuse the developer reading your code.